Integrations/integration-Thinkst_Canary.yml

```diff
@@ -0,0 +1,558 @@
+description: By presenting itself as an apparently benign and legitiamte service(s), the canary draws the attention of
+  unwanted activity. When someone trips one of the Canary's triggers, an alert is sent to notify the responsible
+  parties so that action can be taken before valubale systems in your network are compromised.
+configuration:
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: Canary Server URL
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: API auth token
```

Comment:
please change to `API Authentication Token` add to `detailed instructions` how to get this token